app/src/main/java/org/imozerov/streetartview/ui/catalog/ArtListFragment.kt

```diff
@@ -0,0 +1,82 @@
+package org.imozerov.streetartview.ui.catalog
+
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.support.v7.widget.LinearLayoutManager
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import kotlinx.android.synthetic.main.fragment_art_list.view.*
+import org.imozerov.streetartview.R
+import org.imozerov.streetartview.StreetArtViewApp
+import org.imozerov.streetartview.storage.DataSource
+import org.imozerov.streetartview.ui.interfaces.Filterable
+import org.imozerov.streetartview.ui.model.ArtObjectUi
+import rx.Subscription
+import rx.android.schedulers.AndroidSchedulers
+import javax.inject.Inject
+
+class ArtListFragment : Fragment(), Filterable {
+    private val TAG = "AlertListFragment"
+
+    private var listFilterQuery: String = ""
+    private var fetchSubscription: Subscription? = null
+    private var adapter: ArtListAdapter? = null
+
+    @Inject lateinit var dataSource: DataSource
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+        (activity.application as StreetArtViewApp).storageComponent.inject(this)
+    }
+
+    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
+                              savedInstanceState: Bundle?): View? {
+        val rootView = inflater!!.inflate(R.layout.fragment_art_list, container, false)
+        val context = context
+
+        adapter = ArtListAdapter(getContext(), java.util.ArrayList<ArtObjectUi>())
```

Comment:
I wanted to implicitly show java classes that are used.
Also android studio imports java.util.* so it prevents us from importing Observable from rx. java.util.Observable is suggested instead.